common/lib/xmodule/xmodule/course_module.py

```diff
@@ -901,6 +901,15 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    enable_proctored_exams = Boolean(
+        display_name=_("Enable Proctored Exams"),
+        help=_(
+            "Set this to true if your course would like to support timed proctored exams."
```

Comment:
"Enter true or false. If this value is true, timed and proctored exams are enabled in your course."
(To match existing patterns in Advanced Settings UI text)